netbox/project-static/src/search.ts

```diff
@@ -25,20 +55,26 @@ export function initQuickSearch(): void {
   const clearbtn = document.getElementById("quicksearch_clear") as HTMLAnchorElement;
   if (isTruthy(quicksearch)) {
     quicksearch.addEventListener("keyup", quickSearchEventHandler, {
-      passive: true
-    })
+      passive: true,
+    });
     quicksearch.addEventListener("search", quickSearchEventHandler, {
-      passive: true
-    })
+      passive: true,
+    });
+    quicksearch.addEventListener("change", handleQuickSearchParams);
+
     if (isTruthy(clearbtn)) {
-      clearbtn.addEventListener("click", async () => {
-        const search = new Event('search');
-        quicksearch.value = '';
-        await new Promise(f => setTimeout(f, 100));
-        quicksearch.dispatchEvent(search);
-      }, {
-        passive: true
-      })
+      clearbtn.addEventListener(
+        "click",
+        async () => {
+          const search = new Event("search");
+          quicksearch.value = "";
+          await new Promise(f => setTimeout(f, 100));
+          quicksearch.dispatchEvent(search);
+        },
+        {
+          passive: true,
+        },
+      );
```

Comment:
Please revert any changes that aren't pertinent to the new functionality to limit the scope of PR reviews. (If cleanup is needed, we can tackle those items in a separate housekeeping issue.)